pkg/download/automation/automation.go

```diff
@@ -64,39 +64,44 @@ func (d *Downloader) Download(projectName string, automationTypes ...config.Auto
 
 	configsPerType := make(v2.ConfigsPerType)
 	for _, at := range automationTypes {
+		lg := log.WithFields(field.Type(at.Resource))
```

Comment:
nit/note: This actually increased the memory footprint of the logger, as you now wrap the log core twice, once for the resource filed, once again for the extra "WithFields()" every log message uses. 
WithFields(<type>, <error>, <...>) will only be one wrap

Reply:
Is this noticeable that we should work on that? As I see it, the loggers here are so short-lived that it should not be an issue, right?

Reply:
More a question of why you moved it into a shared partially initialized logger struct. If you did it due to the overhead of extra logger cores (which we had some issues with in the past), it would not actually have that effect.
If just for personal preference of structure, it's perfectly fine here IMO. 
Thus a nit/note, just to let you know :)